messagee_box/MessageeBox.py

```diff
@@ -0,0 +1,19 @@
+import tkinter as tk
+from tkinter import messagebox
+
+class MessageBox:
+    """Class, which represents message box object"""
+
+    def __init__(self, text, title):
```

Comment:
Here you should also change its image, it's not an information, so bubble is not appropriate object.